Wikipedia/assets/codemirror/resources/addon/search/search.js

```diff
@@ -137,20 +137,14 @@
         }
       }
 
-      const focusedMatchID = `cm-searching-focus-id-${focusedMatchIndex}`;
```

Comment:
oh were the removals from this file deliberate? they seem unrelated to text size...

Reply:
They were changes related to find in page that were brought into www but not built into the assets folder